op-challenger/game/types/types.go

```diff
@@ -33,3 +35,9 @@ func GameStatusFromUint8(i uint8) (GameStatus, error) {
 	}
 	return GameStatus(i), nil
 }
+
+type GameData struct {
```

Comment:
nit: GameMetadata could be a better name.

Reply:
Yeah good call, I'll change that.